README.md

```diff
@@ -12,7 +12,7 @@
 [![PyPi][downloads-shield]][downloads-url]
 [![Contributors][contributors-shield]][contributors-url]
 [![License][license-shield]][license-url]
-[![Build][build-shield]][build-url]
+<!-- [![Build][build-shield]][build-url] -->
```

Comment:
what does this do?

Reply:
It removes the `Build: Failing` badge on our Readme. This currently points to some CI workflow on develop which is just too flaky, not sure why that should be displayed on the `main` branch Readme which is always passing as it's pointing to a release.